SolutionInfo.cs

```diff
@@ -1,10 +1,12 @@
 using System.Runtime.CompilerServices;
 using System.Reflection;
 using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
 [assembly: AssemblyCompany("Open Systems Pharmacology Community")]
 [assembly: AssemblyProduct("OSPSuite.DataBinding")]
 [assembly: AssemblyCopyright("Copyright � 2002-2019 - Open Systems Pharmacology Community")]
 [assembly: ComVisible(false)]
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
 [assembly: InternalsVisibleTo("OSPSuite.DataBinding.Tests")]
+[assembly: SupportedOSPlatform("windows")]
```

Comment:
We need this to tell VS that all assemblies are targeting windows only, otherwise there is a lot of warnings generated.